status-resolver/build.gradle

```diff
@@ -23,11 +24,22 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+}
+
+ext {
+    kotlin_coroutines_version = "1.3.7"
+}
+
 dependencies {
-    testImplementation 'junit:junit:4.12'
-    implementation 'androidx.appcompat:appcompat:1.0.0'
-    api 'io.reactivex.rxjava2:rxjava:2.2.6'
-    api 'io.reactivex.rxjava2:rxandroid:2.1.0'
+    testImplementation 'junit:junit:4.13'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation "androidx.core:core-ktx:+"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlin_coroutines_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlin_coroutines_version"
+    testImplementation "com.google.truth:truth:1.0.1"
```

Comment:
group test implementations together pls (in each build.gradle)

Reply:
1625e84cb028ce7b65cc6bd34201660fe27780f8